CHANGELOG.md

```diff
@@ -1,8 +1,28 @@
 # go-ipfs changelog
 
-## v0.10.0 TBD
+## v0.10.0 2021-09-30
 
-**IPLD Levels Up**
+We're happy to announce go-ipfs 0.10.0. This release brings some big changes to the IPLD internals of go-ipfs that making working with non-UnixFS DAGs easier than ever. There are also a variety of new commands and configuration options available.
+
+As usual, this release includes important fixes, some of which may be critical for security. Unless the fix addresses a bug being exploited in the wild, the fix will _not_ be called out in the release notes. Please make sure to update ASAP. See our [release process](https://github.com/ipfs/go-ipfs/tree/master/docs/releases.md#security-fix-policy) for details.
+
+### 🛠 TLDR: BREAKING CHANGES
+
+- `ipfs dag get`
+  - default output changed to  [`dag-json`](https://ipld.io/specs/codecs/dag-json/spec/)
+  - unixfs (and dag-pb) field names changed (impacts userland code that works with `dag-pb` objects returned by this `dag get`)
```

Comment:
This line and the one below it should match up. I'm thinking it's more accurate to be
```suggestion
  - dag-pb (e.g. unixfs) field names changed - impacts userland code that works with `dag-pb` objects returned by this `dag get`
```

IIRC there's nothing special about how the changes that effect UnixFS, they're just dag-pb changes. Sound right @willscott?